### DISCUSSION TITLE
Welcome to textCleanup Discussions!

### DISCUSSION THREAD

**Discussion by develiza** [develiza/textCleanup#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by develiza** [comment]
- add some intro, explaining what the tool does
- further develop tool, adding possibility to download txt file

**Reply by bethha** [reply]
add some footer with datentschutz stuff

**Reply by bethha** [reply]
do research on how to add datenschutz to github pages site
